### ISSUE TITLE
Web Workers sample for Playground

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#319]
```
This sample is quite challenging given the architecture of the playground which 
is meant to show only one live snippet at a time, whereas web workers have also 
code in the file containing the worker itself.

However, there is a recent implementation to have the code of the worker inline 
which would make it easier for us to write this sample.

Assigning to Hanrui for further investigation.

Other references:
http://www.html5rocks.com/tutorials/workers/basics/
http://slides.html5rocks.com/#slide15 (slide titled 'Web Workers')
```

Original issue reported on code.google.com by `erne...@google.com` on 1 Dec 2010 at 12:36

**Comment by GoogleCodeExporter** [comment]
```
Specific chapter about inline web workers: 
http://www.html5rocks.com/tutorials/workers/basics/#toc-inlineworkers
```

Original comment by `erne...@google.com` on 1 Dec 2010 at 12:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 8 Dec 2010 at 1:12
- Added labels: **Milestone-4.2**
- Removed labels: **Milestone-4.1**

**Comment by GoogleCodeExporter** [comment]
```
Revision 
349b2db4db(http://code.google.com/p/html5rocks/source/detail?r=349b2db4dbcc1d7d8
349d7d80c1de1705c11f6e5) fixed this.
```

Original comment by `han...@html5rocks.com` on 20 Dec 2010 at 4:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
The inline Web Worker demo is not working well when you switch from HTML editor 
to JS editor. We extract all codes between <script> and </script> into JS 
editor but this logic is not right for inline Web Worker demo.
I will fix it.
```

Original comment by `han...@google.com` on 13 Apr 2011 at 6:06
- Changed state: **Accepted**
- Added labels: **Milestone-5.5**
- Removed labels: **Milestone-4.2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 47ef720783bd: 
http://code.google.com/p/html5rocks/source/detail?r=47ef720783bda5bb041b40c57a24
294d0cb020ce
```

Original comment by `han...@google.com` on 13 Apr 2011 at 9:53
- Changed state: **Fixed**
